app/controllers/DeparturesController.scala

```diff
@@ -64,6 +67,7 @@ class DeparturesController @Inject()(cc: ControllerComponents,
                 .submitMessage(departure.departureId, departure.nextMessageCorrelationId, message, DepartureStatus.DepartureSubmitted)
                 .map {
                   case SubmissionProcessingResult.SubmissionSuccess =>
+                    auditService.auditEvent(AuditType.User.DepartureDeclarationSubmitted, request.body)
```

Comment:
Is this the best location? Do we not want to log it before we send it so that the two audit messages can be used to understand if something went wrong in-flight?